### PR TITLE
May 2023 post

### PR DESCRIPTION
**Pull request by YOU54F**
Main Blog - https://deploy-preview-264--docs-pact-io.netlify.app/blog/2023/05/25/pact-open-source-update-may-2023

New section - Community Corner - Adam Cox - https://deploy-preview-264--docs-pact-io.netlify.app/users/community_corner/adam_cox_interview_may2023

New Contributing Section - Ci/CD Systems https://deploy-preview-264--docs-pact-io.netlify.app/contributing/ci

New Guide - Pact Ecosystems - https://deploy-preview-264--docs-pact-io.netlify.app/diagrams/ecosystem

### CODE REVIEW

File: website/blog/2023-05-25-pact-open-source-update-may-2023.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+As the `pact_broker` nears its birthday, it decided that is better get ready. You'll now find the latest `pact_broker` and `pact_broker-docker` projects have been upgrades to support Ruby `3.2`.
+
+You'll be able to pull multi-arch builds for `arm/arm64` machines soon, by adding the `-multi` flag to your `pact_broker` tag.
+
+### Pact FFI
+
+### Pact Plugins
+
+## Pact Ecosystems
```
Comment (mefellows) [inline website/blog/2023-05-25-pact-open-source-update-may-2023.md:62]: This whole section (and the linked ecosystem page is 🙌 )

File: website/blog/2023-05-25-pact-open-source-update-may-2023.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+tags: [pact, oss, community]
+keywords: [pact, oss, community]
+date: 2023-05-25
+draft: false
+hide_table_of_contents: false
+---
+
+It's May, the sunshine ☀️ is upon us, and you might be reading this in the park 🏞️. If you are, watch out, theres a cyclist 🚴coming. 🛎️ 🛎️
+
+Phew, that was close. Right, well now we are sorted, we've got some goodies lined up for this month, as the city of open-source never sleeps.
```
Comment (mefellows) [inline website/blog/2023-05-25-pact-open-source-update-may-2023.md:14]: 😆 .

File: website/blog/2023-05-25-pact-open-source-update-may-2023.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+### Pact-Broker
+
+As the `pact_broker` nears its birthday, it decided that is better get ready. You'll now find the latest `pact_broker` and `pact_broker-docker` projects have been upgrades to support Ruby `3.2`.
+
+You'll be able to pull multi-arch builds for `arm/arm64` machines soon, by adding the `-multi` flag to your `pact_broker` tag.
+
+### Pact FFI
+
+### Pact Plugins
```
Comment (mefellows) [inline website/blog/2023-05-25-pact-open-source-update-may-2023.md:60]: The JVM now supports auto-loading plugins as of https://github.com/pact-foundation/pact-plugins/commit/59cd7126c5d72490fae671a953fc824cff8cd5d5.

I think it's not in rust yet, right @uglyog ?

Comment (mefellows) [reply website/blog/2023-05-25-pact-open-source-update-may-2023.md:60]: Actually, it looks like it was added. I can't see when, but https://github.com/pact-foundation/pact-plugins/commit/639caebdbfb9abb05f51f570900462ea0b88204e implies it's done.

File: website/blog/2023-05-25-pact-open-source-update-may-2023.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  - `npm install @pact-foundation/pact --ignore-scripts`
+  - Not require a full developer tool-chain included Python to run `node-gyp` in order to generate the required `node` bindings. We'll do the grunt work, so you don't have to.
+
+### Pact-Broker
+
+As the `pact_broker` nears its birthday, it decided that is better get ready. You'll now find the latest `pact_broker` and `pact_broker-docker` projects have been upgrades to support Ruby `3.2`.
+
+You'll be able to pull multi-arch builds for `arm/arm64` machines soon, by adding the `-multi` flag to your `pact_broker` tag.
+
+### Pact FFI
```
Comment (mefellows) [inline website/blog/2023-05-25-pact-open-source-update-may-2023.md:58]: Lots of activity in https://github.com/pact-foundation/pact-reference/blob/master/rust/pact_ffi/CHANGELOG.md#044---bugfix-release.

File: website/blog/2023-05-25-pact-open-source-update-may-2023.md (hunk 3 above)
Comment (rholshausen) [reply website/blog/2023-05-25-pact-open-source-update-may-2023.md:60]: Of course it is in Rust, that was added first. JVM was added last.

**Comment by mefellows** [comment]
Worth a shout out for Erlang? https://pact-foundation.slack.com/archives/C5F4KFKR8/p1684648796087529?

**Comment by YOU54F** [comment]
yeah 100%On 21 May 2023, at 10:03, Matt Fellows ***@***.***> wrote:﻿
Worth a shout out for Erlang? https://pact-foundation.slack.com/archives/C5F4KFKR8/p1684648796087529?

**Comment by YOU54F** [comment]
oh nice will mention both!On 21 May 2023, at 10:03, Matt Fellows ***@***.***> wrote:﻿
Worth a shout out for Erlang? https://pact-foundation.slack.com/archives/C5F4KFKR8/p1684648796087529?

**Pull request merged**